bentoml/_internal/frameworks/keras.py

```diff
@@ -188,8 +182,9 @@ def save(
     model: "keras.Model",
     *,
     save_format: t.Optional[str] = "tf",
-    custom_objects: t.Optional[t.Dict[str, t.Any]] = None,
     store_as_json_and_weights: t.Optional[bool] = False,
+    labels: t.Optional[t.Dict[str, str]] = None,
+    custom_objects: t.Optional[t.Dict[str, t.Any]] = None,
```

Comment:
should we differentiate Keras custom_objects with our model store custom_objects?